packages/common/src/audius-query/types.ts

```diff
@@ -140,4 +135,14 @@ export type QueryHookResults<Data> = {
   data: Data
   status: Status
   errorMessage?: string
+  forceRefresh: () => void
 }
+
+export type MutationHookResults<Args, Data> = [
+  (fetchArgs: Args, options?: QueryHookOptions) => void,
```

Comment:
I think it makes sense to keep this one separate from the results object since it serves as a good reminder that the invocation pattern is different than query hooks

This way the type system will effectively warn you if you're using mutation when you expect query behavior or vice-versa